### ISSUE TITLE
There is an edge that says: Autoimmune Disease has_phenotype Addison Disease with no evidence. "Disease has_phenotype Disease" is likely not a valid statement. Any chance we could drill down a bit more to find out which source we can contact to help?

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#3]
## Steps to reproduce:
search for Addison Disease, click on Zenapax

## Screenshots:
![screenshot](https://user-images.githubusercontent.com/4561292/189750763-2feaefc8-1aff-41d6-b9a5-7ecfa0592581.png)

![screenshot](https://user-images.githubusercontent.com/4561292/189751092-415c191a-fec1-42a3-aadb-dc01935a993f.png)

**Comment by sstemann** [comment]
@sierra-moxon @gaurav i was trying to retest this one (by running a new query using Addison Disease) but it's not showing up in the UI Disease Selector.  

![image](https://user-images.githubusercontent.com/38321826/223854332-3bacd027-21d9-4293-b985-44796aa9665b.png)

SO, I looked at the query sent in this PK and determined the MONDO used was MONDO:0009410.

That MONDO still looks valid https://monarchinitiative.org/disease/MONDO:0009410

But I put it in Node Norm and I don't get anything
![image](https://user-images.githubusercontent.com/38321826/223854020-cfb20bfa-46d4-40ea-9b47-bfb4587ebe6a.png)

I looked at Name Resolver and put in Addison Disease, it did return with MONDOs but not 9410.  So maybe this is a different issue, the title is looking for evidence, I wanted to see if we're still returning this edge, but I am a bit stuck
